### PR TITLE
#5698 templates/*: replace faq html with styleguid and rm gruenbuchfaq temp…

### LINKED ISSUE #278: Default background colour for blocks
**URL:** https://digitalstrategie-dev.liqd.net/%C3%BCber-den-prozess-overview-page/
**user:** visitor
**expected behaviour:** white background for all blocks when no colour chosen
**behaviour:** background is grey
**important screensize:**
**device & browser:** Macbook firefox, iphone safari
**Comment/Question:** 
Tested block: teaser centered, theses, quote, faq

Screenshot?
grafik

### PR DESCRIPTION
**Pull request by phillimorland**
…late and update css

fixes  #278

### CODE REVIEW

File: digitalstrategie/assets/scss/components/_ds-block.scss [hunk 1]
```diff
@@ -15,55 +15,35 @@
...
         .background-image + .content-container *:not(.button),
         // richtext and cta centered styling
         .button,
         h2,
         h3,
+        h3:after,
         h4 {
             background-color: $green;
-            color: $body-bg;
+            color: $body-bg !important;
```
Comment (khamui) [inline digitalstrategie/assets/scss/components/_ds-block.scss:27]: just saw that the lines between the FAQuestions are differently thick? can you replicate it?
And the top padding/margin is quite big.

let me know if you want to tackle it now or i add issues?

**Review by khamui — COMMENTED** [review]
some style questions.
btw currently Gruenbuch Detail Page isn't working. Will be fixed once working on that story.

File: digitalstrategie/assets/scss/components/_ds-block.scss (hunk 1 above)
Comment (phillimorland) [reply digitalstrategie/assets/scss/components/_ds-block.scss:27]: @khamui can you send me screenshots? it was looking like the styleguide on my machine

Comment (khamui) [reply digitalstrategie/assets/scss/components/_ds-block.scss:27]: update: on my macbook screen it looks fine. merging and doublecheck when people testing

**Pull request merged**